src/uct/ib/rc/base/rc_iface.c

```diff
@@ -556,6 +560,24 @@ uct_rc_iface_init_max_rd_atomic(uct_rc_iface_t *iface,
     return UCS_OK;
 }
 
+static void
+uct_rc_iface_set_flush_remote(uct_rc_iface_t *iface,
+                              const uct_rc_iface_common_config_t *config,
+                              const uct_iface_params_t *params)
+{
+    uct_ib_md_t *md = ucs_derived_of(iface->super.super.md, uct_ib_md_t);
+
+    if ((md->cap_flags & UCT_MD_FLAG_FLUSH_REMOTE) && config->flush_remote &&
+        UCT_IFACE_PARAM_FEATURE(params, GET) &&
```

Comment:
why need to require GET?

Reply:
> why need to require GET?

GET is used for flush_remote op itself. removed